src/client/routes/saksbilde/utbetaling/Utbetaling.tsx

```diff
@@ -22,22 +26,32 @@ const FeilmeldingContainer = styled.div`
     margin-top: 1rem;
 `;
 
-const revurderingEnabled = (person: Person, periode: Tidslinjeperiode): boolean =>
-    overstyreUtbetaltPeriodeEnabled &&
-    person.arbeidsgivere.length > 0 &&
-    periode === person.arbeidsgivere[0].tidslinjeperioder?.[0]?.[0] &&
-    [Tidslinjetilstand.Utbetalt, Tidslinjetilstand.UtbetaltAutomatisk].includes(periode.tilstand);
+const førsteArbeidsgiversSistePeriode = (person: Person) => person.arbeidsgivere[0].tidslinjeperioder?.[0]?.[0];
+
+const kunEnArbeidsgiver = (person: Person) => person.arbeidsgivere.length === 1;
+export const revurderingEnabled = (
+    person: Person,
+    periode: Tidslinjeperiode,
+    kanRevurderingRevurderes: boolean
+): boolean => {
+    return (
+        overstyreUtbetaltPeriodeEnabled &&
+        kunEnArbeidsgiver(person) &&
+        periode === førsteArbeidsgiversSistePeriode(person) &&
+        ([Tidslinjetilstand.Utbetalt, Tidslinjetilstand.UtbetaltAutomatisk].includes(periode.tilstand) ||
+            (periode.tilstand === Tidslinjetilstand.Revurdert && kanRevurderingRevurderes))
+    );
+};
 
 const overstyringEnabled = (person: Person, periode: Tidslinjeperiode): boolean =>
     overstyrbareTabellerEnabled &&
-    person.arbeidsgivere.length === 1 &&
-    ([
+    kunEnArbeidsgiver(person) &&
```

Comment:
Syns `person.arbeidsgivere.length === 1` er minst like lesbart som `kunEnArbeidsgiver(person)`. Trenger vi denne?

Reply:
Hvis vi trenger å sjekke om personen har kun én arbeidsgiver flere steder i speil tenker jeg at det er fint med en funksjon for det. Da kan den kanskje ligge et annet sted, som f.eks. `src/client/mapping/selectors.ts`.